README.md

```diff
@@ -771,6 +771,19 @@ being reset and all new connections to the system are rejected.  Existing
 connections will be unaffected. Applying changes with this option in production
 might cause temporary service failures with new connections during the operation.
 
+### system_is_transactional
```

Comment:
Variables must be prefixed with `storage_`

Reply:
Is there some way to determine if a system is transactional?

Reply:
Would it be better to use transactional_update_ or __firewall_is_transactional as a prefix instead? This way, it will be clear that the variables and tasks are related to transactional update systems rather than storage systems. 

One way to determine would be to check for /sbin/transactional-update stat similar to the ostree tasks, or would it be better to keep it as user-defined? -

~~~
 - name: Determine if system is transactional update and set flag
      when: not __firewall_is_transactional is defined
      block:
        - name: Check if transactional-update exists in /sbin
          stat:
~~~
~~~
    - name: Handle reboot for transactional update systems
      when:
        - __firewall_is_transactional | d(false)
        - firewall_package_result is changed
~~~

Reply:
For example, the role determines if the system is an rpm-ostree based system by looking for the file /run/ostree-booted and setting some variables - https://github.com/linux-system-roles/firewall/blob/main/tasks/firewalld.yml#L13

But perhaps this case is different?  I know that on rpm-ostree systems, you can use the `rpm-ostree` command to stage packages for installation, then reboot the system in order to apply the changes.  Perhaps your transactional system is similar? and there is no way for the role to determine if the system is transactional?